src/cmap/cmap.go

```diff
@@ -71,6 +71,10 @@ func (m *Map[K, V]) Get(key K) V {
 	return v
 }
 
+func (m *Map[K, V]) Contains(key K) bool {
```

Comment:
As noted irl, this is a potential performance hit as we'll do this for every build target, but for now it's a price we're willing to pay. We'll keep an eye on the performance test